src/controllers/taskInstanceController.js

```diff
@@ -0,0 +1,23 @@
+function taskInstanceController({ taskInstanceService, httpErrors }) {
+  async function getAll(req, res) {
+    const tasks = await taskInstanceService.getAll();
+    return res.json(JSON.parse(tasks));
```

Comment:
Did you try just doing `res.json(tasks)`? I'm wondering if `JSON.parse` is needed here.

Reply:
I moved this to the repository as I´m receiving string from the database.